_events/2016-01-31-feminism-is-for-everybody.md

```diff
@@ -0,0 +1,9 @@
+---
+identifier: feminism-is-for-everybody
+slug: feminism-is-for-everybody
+title: "January Book Club"
+date: "2017-01-31 14:00:00"
```

Comment:
Yeah it seems `date` is a key used in the `Jekyll` default implementation that is actually a `datetime` and is used as a predefined variable on a `post`.

That said, we're not using the out-of-the-box `post` functionality, so I move to update this to `datetime` to reflect the thing that it is. :-)